pnpm-lock.yaml

```diff
@@ -5387,7 +5403,7 @@ packages:
     engines:
       node: '>=8'
     resolution:
-      integrity: sha512-HGyxoOTYUyCM6stUe6EJgnd4EoewAI7zMdfqO+kGjnlZmBDz/cR5pf8r/cR4Wq60sL/p0IkcjUEEPwS3GFrIyw==
+      integrity: sha1-t5hCCtvrHego2ErNii4j0+/oXo0=
```

Comment:
why were the checksums downgraded to sha1? do you use some old Node.js version or a non-standard registry?

Reply:
not sure, running node `v12.18.3`, npm `6.14.6` and pnpm `5.13.6`

I was using a different registry, and [this](https://github.com/pnpm/pnpm/pull/3035/commits/a363cb066b4dfa4613519cfb5504fd7dd2fbc102) is the change after running everything again with `npm.js`

maybe its a local setup?